.genx/tests/fixtures/testapp/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.3'
+services:
+  gsf:
+    tty: true
+    container_name: gsf
+    build:
+      context: server/jvm/
+      dockerfile: Dockerfile
+    networks:
+      qs_network:
+        aliases:
+          - localnode
+    volumes:
+      - ./server/jvm/testapp-site-specific/src/main/resources/cfg/:/home/testapp/run/site-specific/cfg
```

Comment:
question:

just curious with `/cfg/:/home` vs `/cfg:/home/` (both happening here)... is that correct?